src/utilities/xml/XMLValidator.cpp

```diff
@@ -140,7 +140,7 @@ openstudio::path XMLValidator::schematronToXslt(const openstudio::path& schemaPa
   saveXmlDocToFile(xsltPath, compiled);
 
   xmlFreeDoc(compiled);
-  LOG(Info, "Saved transformed XSLT Stylesheet at '" << toString(xsltPath) << "'.");
+  LOG(Trace, "Saved transformed XSLT Stylesheet at '" << toString(xsltPath) << "'.");
```

Comment:
e7f46c8dd52d1855e3891b71c2247b0478f2a0d0 is optional, but I'm sick and tired of seeing too much verbosity in XMLValidator (which I'm responsible for, mind you), so I downgraded every Debug/Info message to a Trace.

You shouldn't have to think about what it's doing, and if you do need to debug something, just use Trace